### PR TITLE
DOC-482: Fix quickstart JavaScript URL

### PR DESCRIPTION
**Pull request by m-hulbert**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

This PR fixes the URL for the JavaScript CDN in the quickstart guide. See [JIRA](https://ably.atlassian.net/browse/DOC-482) for further information.

## Review

To review this PR, visit [this page](http://ably-docs-pr-1196.herokuapp.com/quick-start-guide) and view the page in JavaScript.

### CODE REVIEW

File: content/root/quick-start-guide.textile [hunk 1]
```diff
@@ -489,6 +489,7 @@ This guide introduced the basic concepts of Ably and illustrated how easy it is
 * Find out more about the "Realtime Client Library SDK":/realtime.
 * Find out more about the "REST Client Library SDK":/rest.
 * Try out some "tutorials":https://ably.com/tutorials.
+* Provision Ably applications using the "Control API":/control-api.
```
Comment (marklewin) [inline content/root/quick-start-guide.textile:492]: Would it be worth changing the absolute ref to tutorials here, too?

Comment (tbedford) [reply content/root/quick-start-guide.textile:492]: > Provision Ably applications

Minor, but can we change to "Ably apps" for consistency in Control API docs. Thanks.

Comment (m-hulbert) [reply content/root/quick-start-guide.textile]: Changed both, thanks!

**Review by marklewin — APPROVED** [review]
LGTM @m-hulbert !

**Review by tbedford — APPROVED** [review]

**Pull request merged**